File: modules/WebApp/Payload.cs
using System;

namespace WebApp
{
    public class Payload
    {
        public DateTime TimeStamp { get; set; }

        public bool IsAirConditionerOn { get; set; }

        public double Temperature  { get; set; }

        public string TagKey  { get; set; }

    }
}

File: modules/PythonDataGenerator/payload.py
import json

class Payload:
  def __init__(self, timeStamp, isAirConditionerOn, temperature,tagKey):
    self.TimeStamp = timeStamp
    self.IsAirConditionerOn = isAirConditionerOn
    self.Temperature = temperature
    self.IsAirConditionerOn = isAirConditionerOn
    self.TagKey = tagKey

def toJson(self):
    return json.dumps(self.__dict__)
         